Neos.Media/Classes/Domain/Model/ThumbnailConfiguration.php

```diff
@@ -159,6 +166,14 @@ public function getHash()
     }
 
     /**
+     * @return int
```

Comment:
Nah, with scalar type hints `int` is the way to go, so we should use that consistently.